### ISSUE TITLE
simply git command and do not use regex

### ISSUE THREAD

**Issue by bcoe** [conventional-changelog/conventional-changelog#164]
Using gitlab the response I'm getting from git log --tags --simplify-by-decoration --pretty=format:"%d" is:
(v1.0.2)
(v1.0.1)
(v1.0.0)

And that breaks the matching of tags so it will return no tags for this project.

CC: @pjpsoares, @Rayman, @psyrendust, @blowsie let's move this conversation to our new mono-repo 👍

**Comment by blowsie** [comment]
I'm not sure if I'm missing one of the requirements here but this change is working for me

```
var regex = /\(tag:\s*(.+?)[,\)]/gi;
var cmd = 'git log --tags --simplify-by-decoration --pretty=format:"%d"';
```

It seems more robust to me since, it is no longer iterating rows which are not tags.

---

This for me, fixed my problem with git-semver tags not picking up some of my tags.

**Comment by blowsie** [comment]
Is this commit sufficient? 
https://github.com/conventional-changelog/conventional-changelog/compare/master...blowsie:patch-1

**Comment by Tapppi** [comment]
@blowsie Hold on, just passed by here and am wondering.. The old regex parses multiple tags on a row, but the new one doesn't, is this **intended**? Just want to check. Also the name of this issue does not reflect the changes, is the idea to change the git command to increase robustness?

**Comment by blowsie** [comment]
@Tapppi Yes for me the exiting code does not work at all.
I'm using gitlabs and smart git.

I could update my code to handle multiple tags per line, but in my case it is not necessary.

**Comment by Tapppi** [comment]
Yeah im just pointing out that if it gets merged here, that would be probably be required? Would love a PR for it :wink:

**Comment by blowsie** [comment]
So infact, its only the command line that needs to change, the regex can stay as it was.
![image](https://cloud.githubusercontent.com/assets/308572/24108093/8b7b2630-0d84-11e7-9692-70d3045a8842.png)

I can create a PR if we think this is good to go?

**Comment by Tapppi** [comment]
Go ahead! @stevemao can accept it, he knows more about this particular Repo.

**Comment by hbetts** [comment]
Closing this issue as it seems to have been addressed by @blowsie in #192
